packages/grafana-ui/src/components/PieChart/PieChart.tsx

```diff
@@ -130,7 +113,7 @@ export const PieChart: FC<PieChartProps> = ({
     fieldConfig,
     reduceOptions,
     data,
-    theme,
+    theme: theme.v1,
```

Comment:
nit: Maybe the `getLegend` arrow function here should also be a hook?

Reply:
moved this out to a separate function